kong/conf_loader.lua

```diff
@@ -556,29 +567,46 @@ local function load(path, custom_conf)
     -- intermediate Kong config file in the prefix directory
     local mt = { __tostring = function() return "" end }
 
-    conf.proxy_listeners, err = parse_listeners(conf.proxy_listen)
+    conf.proxy_listeners, err = parse_listeners(conf.proxy_listen, { "mock" })
```

Comment:
why adding the "mock" flag here as an extra argument?, why not in the `flags` list above @398?

Reply:
Because we don't want `mock` flag to `admin` flags.

Reply:
yes, figured that by now. In that case we need to take out "proxy_protocol" in a similar way. As you can now apply that on the admin interface.